content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md

```diff
@@ -21,24 +21,76 @@ table th:nth-of-type(2) {
-|                                              | **Upgrading a CPI** - A new PUT method has been added to the `virtualnode` resource to upgrade a virtual node's CPI: `/api/v1/virtualnode/{virtualNodeShortHash}/cpi/{target-CPI-file-checksum}`. Before upgrading, the virtual node must be in maintenance mode with no other operations currently in progress. You can check the list of running flows using `GET /api/v1/flow/{virtualNodeShortHash}`. When the virtual node is in maintenance, and when no flows are running (that is, all flows have either "COMPLETED", "FAILED" or "KILLED" status), it is safe to trigger a virtual node upgrade. The target CPI should have the same name, signer summary hash, and MGM group ID as the existing CPI. 
-|                                              | **Corda CLI** - Any worker and CLI long options that used camel case now use kebab case. This may require that existing scripts are updated. |
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:"Poppins", -apple-system, blinkmacsystemfont; font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:"Poppins", -apple-system, blinkmacsystemfont; font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-o7ty{background-color:#FFF;border-color:#c0c0c0;color:#1D2343;font-weight:bold;text-align:left;vertical-align:top}
+.tg .tg-7ewn{background-color:#efefef;border-color:#c0c0c0;color:#1D2343;font-weight:bold;text-align:left;width: 30%;vertical-align:bottom}
+</style>
+<table>
+<thead>
+  <tr>
+    <th class="tg-7ewn">Domain</th>
+    <th class="tg-7ewn">Specific Change</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-o7ty" rowspan="4"><span style="font-weight:bolder">Cluster Administration</span></td>
```

Comment:
Is there a way to use some newlines and indentations such that whenever we want to edit this table it would be easier and clearer ?
For example:
```
<td class="tg-o7ty" rowspan="4">
  <span style="font-weight:bolder">Cluster Administration</span>
</td>
```